java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture1/tasks/_6/Number6.java

```diff
@@ -0,0 +1,72 @@
+package com.zoolatech.lecture1.tasks._6;
+
+import java.util.Scanner;
+
+/**Write a program that accepts two numbers and a symbol of an operation
+ * (as a character) and outputs the result of the selected operation.
+ * Possible characters for the operation: ‘+’, ‘-’, ‘*’, ‘/’, ‘%’.
+ * The program should work only with integer values.
+ * Create two versions of the program:
+ * one should use the if-else statement, second - the switch statement.
+ */
+
+public class Number6 {
+    public static void main(String[] args) {
+        Number6 number6 = new Number6();
+        System.out.println(number6.calculatorIfElseVersion());
+        number6.calculatorSwitchCaseVersion();
+
+    }
+
+    private String calculatorIfElseVersion() {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Type First Number:");
+        int number1 = scanner.nextInt();
+        System.out.println("Type Second Number:");
+        int number2 = scanner.nextInt();
+        System.out.println("Select operation:");
+
+        char operation = scanner.next().charAt(0);
+        char addition = '+';
+        char subtraction = '-';
+        char multiplication = '*';
+        char division = '/';
+        char modulus = '%';
+
+        if (operation == addition) {
+            return "Result:" + (number1 + number2);
```

Comment:
it's also possible to simplify this method even more: it can just return a result of an operation, and the phrase `"Result:" + ` can be defined in the main method like this:

```
 System.out.println("Result:" + number6.calculatorIfElseVersion());
```